CRM/Member/BAO/Query.php

```diff
@@ -312,6 +312,18 @@ public static function whereClauseSingle(&$values, &$query) {
         }
         $query->_tables['civicrm_membership'] = $query->_whereTables['civicrm_membership'] = 1;
         return;
+
+      case 'member_override':
```

Comment:
I'm a bit on the fence as to whether we should also be ensuring the xml has the unique_name of membership_is_test - it is definitely consistent with ... something to add it - but we never quite decided if the convention had value or not.